api/api.js

```diff
@@ -0,0 +1,101 @@
+const request = require('request');
```

Comment:
Let's fix all the linter errors and warnings first. You can go to `Files changed` tab on github or run `npm run lint` locally.

Reply:
On it